modules/otel-config-instrumentation.adoc

```diff
@@ -6,7 +6,7 @@
 [id="otel-instrumentation-config_{context}"]
 = OpenTelemetry instrumentation configuration options
 
-The {OTELName} can inject and configure the OpenTelemetry auto-instrumentation libraries into your workloads. Currently, the project supports injection of the instrumentation libraries from Go, Java, Node.js, Python, .NET, and the Apache HTTP Server (`httpd`).
+The {OTELName} injects and configures the OpenTelemetry auto-instrumentation libraries into your workloads. Currently, it supports injecting instrumentation libraries for Go, Java, Node.js, Python, .NET, and the Apache HTTP Server (`httpd`).
```

Comment:
```suggestion
The {OTELName} injects and configures the OpenTelemetry auto-instrumentation libraries into your workloads. Currently, the Operator supports injecting instrumentation libraries for Go, Java, Node.js, Python, .NET, and the Apache HTTP Server (`httpd`).
```
I would use the noun instead of "it" here.

Reply:
Thank you. Updated.